src/codec_svt.c

```diff
@@ -73,7 +73,7 @@ static avifResult svtCodecEncodeImage(avifCodec * codec,
         }
         svt_config->encoder_color_format = color_format;
         svt_config->encoder_bit_depth = (uint8_t)image->depth;
-#if SVT_VERSION_PATCHLEVEL < 8
+#if SVT_VERSION < 0.8.8
```

Comment:
I just wanted to clarify a point. In my previous comment, I wrote

> In pseudo code, the correct check is
> 
> `#if SVT_VERSION < 0.8.8`

Note that `#if SVT_VERSION < 0.8.8` is pseudo code. We cannot use `#if SVT_VERSION < 0.8.8` directly.

There are two common ways to do this check correctly. The first way is to write a complicated expression to check `SVT_VERSION_MAJOR`, `SVT_VERSION_MINOR,` and `SVT_VERSION_PATCHLEVEL`. For example

```
#if SVT_VERSION_MAJOR == 0 && \
  (SVT_VERSION_MINOR < 8 || \
   (SVT_VERSION_MINOR == 8 && \
    SVT_VERSION_PATCHLEVEL < 8))
```

The second way is to pack `SVT_VERSION_MAJOR`, `SVT_VERSION_MINOR`, and `SVT_VERSION_PATCHLEVEL` into a packed value, e.g.,

```
#define SVT_VERSION \
  (SVT_VERSION_MAJOR * 10000 + SVT_VERSION_MINOR * 100 + SVT_VERSION_PATCHLEVEL)
```

and then check the packed value, e.g.,

`#if SVT_VERSION < 808`

Reply:
It would be best to add either the SVT_CHECK_VERSION macro (which is in [SVT-VP9](https://github.com/OpenVisualCloud/SVT-VP9/blob/master/Source/API/EbApiVersion.h.in)) or the packed-version SVT_VERSION macro to SVT-AV1 first, and then libavif can use the new macro in src/codec_svt.c, providing its own backup definition of the new macro for older versions of SVT-AV1.